multi-tenancy-observability/README.md

```diff
@@ -79,61 +100,60 @@ Below an example of OAuth token content coming from Dex:
 }
 ```
 
-The component `loki-multi-tenant-proxy` should be renamed `grafana-multi-tenant-proxy` to be more generic because it will handle multi-tenancy for logs, metrics, traces, etc.
-That component should have a configuration in which the mapping between groups of people and tenants is defined.
+The component `loki-multi-tenant-proxy` which has now been renamed to `grafana-multi-tenant-proxy` should be configured with the mapping between groups of people and tenants
 
-Below a proposal of such a configuration:
+Below is a proposal of the configuration of the mapping between tenants and their assigned groups from the active directory:
 
 ```yaml
 groups:
 - name: giantswarm-admins
-  tenant_ids:
+  tenants:
   - mc-name
   - wcdev-name
   - wcprod-name
 - name: Developers
-  tenant_ids:
+  tenants:
   - wcdev-name
 - name: Ops
-  tenant_ids:
+  tenants:
   - wcdev-name
   - wcprod-name
 ...
 ```
 
-That configuration should be dynamically created and updated regardless of the data source.
-We are thinking of having an operator to manage that mapping. The operator would be able to reconcile any kind of data source and generate the configuration expected by `grafana-multi-tenant-proxy`.
+The mapping configuration should be dynamically created and updated regardless of the data source by an operator.
+The operator would be able to reconcile any kind of data source and generate the configuration expected by `grafana-multi-tenant-proxy`.
 
 Below is a graph exposing multi-tenancy proposal:
 
 <img src="./assets/future-multi-tenancy.png" width="300" alt="Multi-tenancy proposal on the read path">
 
-### Opened questions
+#### Operator management
+
+`logging-operator` is an existing component responsible for creating a secret for the multi-tenant-proxy. That secret contains a mapping between credentials and tenant (it's currently equivalent to the cluster id).
+
+We will have to either create a new operator to handle multi-tenancy for all observability tools (`observability-multi-tenancy-operator`) or add this feature into the `logging-operator` and rename it to `observability-operator`.
```

Comment:
Sure but I'd like if we make a decision together :D

Reply:
My opinion on this is that we should have a new observability-operator with different controllers that each take care of their own things (1 for multi-tenancy, 1 for logging, and so on) so it would be easier to see where things are going wrong without having to go in 5 different operators. @giantswarm/team-atlas what do you think?